### PR TITLE
gltf tests 

### PR DESCRIPTION
**Pull request by sushantojal**
adding gltf tests.
GearVRf-DCO-1.0-Signed-off-by: Sushant Ojal sushant.o@samsung.com

depend on PR #1873 in the framework repo.

### CODE REVIEW

File: asset-tests/app/src/main/assets/jassimp/gltf/BoomBox/BarramundiFish.gltf~ [hunk 1]
```diff
@@ -0,0 +1,158 @@
+{
```
Comment (liaxim) [inline asset-tests/app/src/main/assets/jassimp/gltf/BoomBox/BarramundiFish.gltf~]: Is this file used?

File: asset-tests/app/src/androidTest/java/org/gearvrf/tester/AssetImportTests.java [hunk 2]
```diff
@@ -528,9 +529,31 @@ public void jassimpSkinningTREX() throws TimeoutException
...
-    public void jassimpBoxGLTF() throws TimeoutException
+    public void jassimpRoughBoomboxGLTF() throws TimeoutException
     {
-        mHandler.loadTestModel(GVRTestUtils.GITHUB_URL + "jassimp/gltf/BoxTextured-glTF/BoxTextured.gltf", 0, 2, "jassimpBoxGLTF");
+        GVRSceneObject light1 = createLight(mTestUtils.getGvrContext(), 1, 1, 1, 1.8f);
+        GVRSceneObject light2 = createLight(mTestUtils.getGvrContext(), 1, 1, 1, -0.8f);
+        mRoot.addChildObject(light1);
+        mRoot.addChildObject(light2);
+
+        mHandler.loadTestModel(GVRTestUtils.GITHUB_URL + "/jassimp/gltf/BoomBox/BoomBox.gltf", 5, 1, "jassimpRoughBoomboxGLTF");
```
Comment (liaxim) [inline asset-tests/app/src/androidTest/java/org/gearvrf/tester/AssetImportTests.java]: I don't think BoomBox.gltf is at that location. I see it under the app/assets folder.

**Comment by liaxim** [comment]
Planning to merge on Thursday.

jassimpGlossWaterBottleGLTF fails with
```
java.lang.AssertionError: expected:<6> but was:<5>
at net.jodah.concurrentunit.Waiter.fail(Waiter.java:205)
at net.jodah.concurrentunit.Waiter.assertEquals(Waiter.java:40)
at org.gearvrf.tester.AssetEventHandler.checkAssetLoaded(AssetEventHandler.java:102)
at org.gearvrf.tester.AssetEventHandler.loadTestModel(AssetEventHandler.java:165)
at org.gearvrf.tester.AssetImportTests.jassimpGlossWaterBottleGLTF(AssetImportTests.java:553)
at java.lang.reflect.Method.invoke(Native Method)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
at android.support.test.internal.statement.UiThreadStatement.evaluate(UiThreadStatement.java:55)
at android.support.test.rule.ActivityTestRule$ActivityStatement.evaluate(ActivityTestRule.java:257)
at org.junit.rules.RunRules.evaluate(RunRules.java:20)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runners.Suite.runChild(Suite.java:128)
at org.junit.runners.Suite.runChild(Suite.java:27)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at android.support.test.internal.runner.TestExecutor.execute(TestExecutor.java:54)
at android.support.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:240)
at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:1967)
```

**Comment by NolaDonato** [comment]
this means the number of textures the test was expecting to be loaded was 6 but it got 5. i think this is an error with the test but i will let Sushant decide.

**Comment by sushantojal** [comment]
I ran that particular test a few times. It sometimes is 5, and 6 at other times. The model is loaded correctly though. Most probably a timing issue. Will debug.

**Pull request merged**